content/css-blog.md

```diff
@@ -0,0 +1,36 @@
+---
+title: When CSS seems scary
+image: /images/writing.jpg
+imageMeta:
+  attribution:
+  attributionLink:
+featured: true
+authors:
+  - anne-greeth-schot-vanherwijnen
+date: Wed Oct 6 2021
+tags:
+  - css
+  - tech
+  - beginner
+---
+
+In this blogpost I want to show you how I tackle CSS. There are so many CSS rules, so many properties and options that it might feel overwhelming. I would recommend you to start small. 
+
+If we take the example of a HTML button element. You might like it the way it is or you might prefer to give it a bit more charm.  
+
+# Tip 1: Open the browser
+Often when I want to (re)design something I open up the dev-tools in the browser by right clicking “inspect element” and I’ll just write my CSS right there. The browser will give you auto-completion and a ton of options and you can see your changes directly.
```

Comment:
```suggestion
Often when I want to (re)design something I open up the dev-tools in the browser by right clicking and choosing “inspect element” and I’ll just write my CSS right there. The browser will give you auto-completion and a ton of options and you can see your changes directly.
```

Reply:
but also... I think depending on who you're aiming this at you might need to explain "and I'll just write my CSS right there" 🤔 you could even put a gif in here to show you right-clicking and editing a bit of CSS in the right place, what do you think?